app/tests/challenges_tests/test_tasks.py

```diff
@@ -54,64 +56,84 @@ def test_challenge_creation_from_request(challenge_request):
 
 @pytest.mark.django_db
 def test_challenge_request_budget_calculation(challenge_request):
-    assert challenge_request.budget["Data storage cost for phase 1"] == round(
-        challenge_request.phase_1_number_of_test_images
-        * challenge_request.average_size_of_test_image_in_mb
-        * settings.CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR
-        / 1000000
-        / 100,
-        ndigits=2,
-    )
-    assert challenge_request.budget["Compute costs for phase 1"] == round(
-        challenge_request.phase_1_number_of_submissions_per_team
-        * challenge_request.expected_number_of_teams
-        * challenge_request.phase_1_number_of_test_images
-        * challenge_request.inference_time_limit_in_minutes
-        * settings.CHALLENGES_COMPUTE_COST_CENTS_PER_HOUR
-        / 60
-        / 100,
-        ndigits=2,
+    assert (
+        challenge_request.budget["Data storage cost for phase 1"]
+        == math.ceil(
+            challenge_request.phase_1_number_of_test_images
+            * challenge_request.average_size_of_test_image_in_mb
+            * settings.CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR
+            / 1000000
+            / 100
+            / 10
+        )
+        * 10

```

Comment:
FYI When writing tests it is better to hardcode the values rather than duplicating the logic. You can protect yourself from any changes to the variables (or even test changes to the variables) by overriding them for the test.